util/reggen/validate.py

```diff
@@ -1597,27 +1615,38 @@ def validate(regs, **kwargs):
     # auto header generation would go here and update autoregs
 
     if 'no_auto_intr_regs' in regs:
-        no_autoi, err = check_bool(regs['no_auto_intr_regs'],
+        no_auto_intr, err = check_bool(regs['no_auto_intr_regs'],
                                    'no_auto_intr_regs')
```

Comment:
Nit: I like the rename, but the line afterwards needs re-indenting.

Reply:
oh yes, forgot to do that after find-replace